graph-proxy/src/graphql/workflows.rs

```diff
@@ -445,3 +445,457 @@ impl WorkflowsQuery {
         Ok(connection)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::path::PathBuf;
+
+    use serde_json::json;
+    use url::Url;
+
+    use super::*;
+    use crate::graphql::root_schema_builder;
+
+    #[tokio::test]
+    async fn single_workflow_query() {
+        let workflow_name = "numpy-benchmark-wdkwj";
+        let proposal_code = "mg";
+        let proposal_number = 36964;
+        let number = 1;
+        let namespace = format!("{}{}-{}", proposal_code, proposal_number, number);
```

Comment:
We should probably use `crate::graphql::Visit` here. The `to_string` of which would produce your `namespace`